src/test/resources/artifacts/ESB/connector/config/Epic.properties

```diff
@@ -0,0 +1,33 @@
+proxyDirectoryRelativePath=/artifacts/ESB/config/proxies/Epic/
+requestDirectoryRelativePath=/artifacts/ESB/config/restRequests/Epic/
+
```

Comment:
Currently we are automating the integration test case to minimize manual modifications. So could you pls do the needful to automate this.